plugin.video.mediathekview/resources/lib/updateFileDownload.py

```diff
@@ -0,0 +1,247 @@
+# -*- coding: utf-8 -*-
```

Comment:
This functionality might be a bit gray I'd say. @Lunatixz can you double check?

Reply:
@enen92 Yea, IMO a little more than gray. This type of behavior is not permitted in Kodi's repository for a few reasons. Mainly a database file is binary which is not permitted. Self-updating a file via download is also not permitted; I assume the database contains a list of online media? This is also not permitted unless through an API or a local (non-binary) file included with the review. If hosted the API would be vetted by the team for legitimate ownership of any copyrighted material.

Reply:
@enen92 
@Lunatixz
Thank you for your comments and review. I totally agree to your concerns and your efforts put into KODI to control the ecosystem is highly appreciated. Before going into some discussion on the technical details, I would like to clear out your concerns on the legitimate ownership. This plugin is part of the [mediathekview](https://mediathekview.de) organization. We do provide public, open and legal links to public Stations in one condensed API. This is explicitly limited to free media only. The Organization and this approach is well known in Germany and our Software and approach is regularly discussed in IT related papers e.g. [legal question](https://tipps.computerbild.de/software/systemsoftware/mediathekview-legal-ist-die-nutzung-erlaubt-621103.html), [Heise](https://www.heise.de/download/product/mediathekview-60659), [Computerbild](https://www.computerbild.de/download/MediathekView-4503190.html), [Pcwelt](https://www.pcwelt.de/downloads/Web-TV-MediathekView-583413.html), [Giga](https://www.giga.de/downloads/mediathekview/), [Wiki](https://de.wikipedia.org/wiki/MediathekView)
Mediathekview highly respects the ownership of intellectual property and does not support piracy or any kind of copyright violations.
Please let me know if this clarified the situation and if it makes sense to go into a technical discussion to align with KODI guidelines.